### PR TITLE
Make StorageClass configurable

### PR DESCRIPTION
**Pull request by lightnet328**
Make it possible to set the StorageClass field in Portus configuration file from externally.
I need to set Storage Class field for using Dynamic Volume Provisioning.

### CODE REVIEW

**Comment by mssola** [comment]
@stefsuse could you take a look at this ?

File: contrib/helm-charts/portus/values.yaml [hunk 1]
```diff
@@ -110,6 +110,7 @@ registry:
     enabled: false
     accessMode: "ReadWriteOnce"
     size: "10Gi"
+    # storageClass: '-'
```
Comment (stefsuse) [inline contrib/helm-charts/portus/values.yaml:113]: uncomment

**Review by stefsuse — CHANGES_REQUESTED** [review]
Please update Chart.yaml version

**Comment by lightnet328** [comment]
I sent some pull requests, how do I update version?

File: contrib/helm-charts/portus/values.yaml (hunk 1 above)
Comment (lightnet328) [reply contrib/helm-charts/portus/values.yaml:113]: I wrote such a description for consistency with other charts.
https://github.com/kubernetes/charts/search?q=storageClass+filename%3Avalues.yaml
I'm not picky about it, should I uncomment it?

Comment (stefsuse) [reply contrib/helm-charts/portus/values.yaml:113]: I see. No, on second thought it is ok as is thanks.

**Comment by stefsuse** [comment]
As this change is also included in #38 please close this PR

**Pull request closed without merging**